collections/esdc-8d-0.083deg-1x2160x4320-2.1.1.yaml

```diff
@@ -0,0 +1,202 @@
+Name: esdc-8d-0.083deg-1x2160x4320-2.1.1
+Description: |
+  esdc-8d-0.083deg-1x2160x4320-2.1.1 is a result of the project Earth System Data Lab, which is now continued as Deep
+  Earth System Data Lab (DeepESDL). It contains relevant variables for Earth System Science, mostly derived from Earth Observation, 
+  but the compilation also includes model or re-analysis data. The DeepESDL project is very grateful to all data owners for kindly 
+  providing the data sets free of charge. They come without any warranty, neither from the owners, 
+  the DeepESDL team or ESA.
+
+Documentation: "[Product information](https://www.earthsystemdatalab.net/index.php/documentation/data-sets/)"
+Image: esdc-8d-0.083deg-1x2160x4320-2.1.1/esdc-8d-0.083deg-1x2160x4320-2.1.1.png
+Explore: "[Open Notebook](https://eurodatacube.com/notebooks/curated/ESDL_EDC_v0.2.ipynb)"
+Resolution: 0.083deg
+GeographicalCoverage: |
+ (179.9569°W, -179.95833°E, -89.95761°S, 89.958336°N)
+ Areas: 
+  - global
```

Comment:
You should remove these rows, since the coordinates already implicate global coverage.

It currently looks like this:

![image](https://user-images.githubusercontent.com/60575896/204853080-1704628d-18e5-4cdb-9baf-f21f0ff0a6c4.png)